### PR TITLE
and page files and router

### PR DESCRIPTION
**Pull request by shivamSspirit**
add page components and all routes

### CODE REVIEW

File: src/Routers/routes.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { BrowserRouter as Router, Route, Routes, Navigate } from "react-router-dom";
+
+// "react-icons": "^4.3.1",
+//     "react-player": "^2.10.0",
+//     "react-router-dom": "^6.2.2",
+//     "react-scripts": "4.0.3",
+//     "react-toastify": "^8.2.0",
+//     "use-local-storage": "^2.3.6",
```
Comment (HeyNitin) [inline src/Routers/routes.js:9]: you can remove this code if it's not required

File: src/Routers/routes.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+// 	<Route
+// 		{...rest}
+// 		/*eslint-disable no-return-assign*/
+// 		render={props => (AuthUtils.isAuthCookiePresent() ? (
+// 			<Component {...props} title={title} />
+// 		) : (
+// 			<Navigate to='/login' replace/>
+// 		))
+// 		}
+// 	/>
```
Comment (HeyNitin) [inline src/Routers/routes.js:36]: you shouldn't publish commented code

**Review by HeyNitin — CHANGES_REQUESTED** [review]
A lot of commented code is there, remove it, rest all good

File: src/Routers/routes.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import PlayListModal from "../components/playlistmanagment/playlistmodal/playlistmodal";
+import PlaylistPage from "../pages/playlistPage/PlaylistPage";
+import Jest from "../pages/jest/jest";
+// import ExploreOne from "../themes/explore-one";
+
+// import * as AuthUtils from '../utils/auth';
+
+// const PrivateRoute = ({ component : Component, title,...rest }) => (
+// 	<Route
+// 		{...rest}
```
Comment (aurobindoGupta) [inline src/Routers/routes.js:28]: remove

File: src/Routers/routes.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+                    <Route exact path="/playlists" element={<PlaylistPage/>}/>
+                    <Route exact path="/jest" element={<Jest/>}/>
+
+                    {/* <Route exact path="/modal" element={<PlayListModal/>}/> */}
+                    
+                    {/* 
+         
+            <Route exact path="/nft/:id" element={<ItemDetails/>} />
+
```
Comment (aurobindoGupta) [inline src/Routers/routes.js:60]: remove

File: src/pages/jest/jest.test.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+// import { helloSUm } from "./jest";
+
+// helloSUm
```
Comment (aurobindoGupta) [inline src/pages/jest/jest.test.js:3]: remove

File: src/pages/landingPage.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Header from '../components/header/Header'
+import Land from '../components/land/Land'
+import Card from '../components/cards/CategoryCard/Card'
+import { useGlobal } from '../context/GlobalContext'
+
+function LandingPage() {
+    const { globalVideos,globalVidCategory } = useGlobal();
+   // console.log('glob', globalVideos)
```
Comment (aurobindoGupta) [inline src/pages/landingPage.js:9]: rremove

**Review by aurobindoGupta — APPROVED** [review]
nice work.
remove commented code.
don't push commented code in pr

**Pull request merged**